azure-pipelines.yml

```diff
@@ -72,7 +79,6 @@ jobs:
         name: py36_test_oldestdeps
 
       - linux: py38-devdeps
-        name: py38_test_devdeps
```

Comment:
Why has this gone?

Reply:
See https://github.com/sunpy/sunpy/pull/4004#discussion_r405604079 TL;DR it isn't used for anything.